.github/workflows/localization_branch_sync.yml

```diff
@@ -0,0 +1,32 @@
+name: Sync Localization Branch
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types: [closed]
+
+jobs:
+  delete_branch:
```

Comment:
Do we really need two jobs? I don't see what are we gaining in creating to instances one that depends on the other one, both run-in a single step and both using ubuntu. I would understand if we were executing steps that need a diff OS. This is not needed.

Reply:
Ah okay I can make it one job